README.md

```diff
@@ -102,6 +102,7 @@ Validator                               | Description
 **isBoolean(str [, options])**          | check if a string is a boolean.<br/>`options` is an object which defaults to `{ loose: false }`. If loose is is set to false, the validator will strictly match ['true', 'false', '0', '1']. If loose is set to true, the validator will also match 'yes', 'no', and will match a valid boolean string of any case. (eg: ['true', 'True', 'TRUE']).
 **isBtcAddress(str)**            | check if the string is a valid BTC address.
 **isByteLength(str [, options])**          | check if the string's length (in UTF-8 bytes) falls in a range.<br/><br/>`options` is an object which defaults to `{min:0, max: undefined}`.
+**isContainerID(str)**                   | check if the string is an ISO6346 shipping container identification.
```

Comment:
I think the name might be a bit too unspecific, as "container ID" feels a bit too generic, e.g. just google it, brought up several hits, like e.g. "docker container ID", "Windows Drivers Container ID", etc.

So I would probably say it makes sense to rename this?

Options could be maybe:
* isShippingContainerID
* isISO6346ContainerID (that one does look ugly to be honest :-D)
* ?

What do you think?

Reply:
We already have some ISO standards so a simple `isISO6346` could also suffice

Reply:
I have changed it to `isISO6346`, for consistency with other validators.